frontend/src/store/Shared.js

```diff
@@ -1,8 +1,13 @@
 
 
 export const getUrl = () => {
-    const {hostname, protocol} = window.location
-    return (hostname === "localhost") ? "http://localhost:8000":`${protocol}/${hostname}`
```

Comment:
No, need to add everything.
Missing extra **/** in the `${protocol}/${hostname}`
Change to `${protocol}//${hostname}`

Reply:
ack